v2/middlewares/consumer/consumer.go

```diff
@@ -0,0 +1,34 @@
+package consumer
+
+import (
+	"context"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+
+	"github.com/honestbank/kp/v2/internal/consumer"
+	"github.com/honestbank/kp/v2/middlewares"
+)
+
+type consumerMiddleware struct {
+	c consumer.Consumer
```

Comment:
```suggestion
	consumer consumer.Consumer
```

I dont like `c.c.XXX()` it hard to read it

Reply:
done :)